pkg/clients/dynatrace/dynatrace_client_test.go

```diff
@@ -174,9 +174,9 @@ func handleRequest(request *http.Request, writer http.ResponseWriter) {
 		(&ipHandler{}).ServeHTTP(writer, request)
 	case "/v1/deployment/installer/agent/connectioninfo":
 		handleCommunicationHosts(request, writer)
-	case "/v1/events":
+	case "/v2/events":
```

Comment:
this endpoint does not exist in v2 as post:
- `/v1/events` to list/post events
- `/v2/events` to list all events
- `/v2/events/ingest` to ingest a custom event

Reply:
thankfully we only POST, so we don't have to care about `/v2/events` 😌 

(quickly added it, still need to test that it actually works)

Reply:
removed it as it requires a different API scope